lib/sync-settings.coffee

```diff
@@ -103,6 +104,11 @@ module.exports =
         atom.notifications.addSuccess "sync-settings: Your settings were successfully backed up. <br/><a href='"+res.html_url+"'>Click here to open your Gist.</a>"
       cb?(err, res)
 
+  viewBackup: ->
+    Shell = require 'shell'
```

Comment:
The usage of this feature is probably to low to benefit from using a global variable...
```
Shell ?= require 'shell'
```